### PR TITLE
Fix "by using" in concepts A--M

### PR DESCRIPTION
**Pull request by Lennonka**
I went slightly mad and started fixing these. Part 1 of many. Please don't hate me.

Renaming of files is done in a separate PR.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12

### CODE REVIEW

File: guides/common/modules/con_composite-content-views-overview.adoc [hunk 1]
```diff
@@ -52,4 +52,4 @@ You can also select the *Auto Publish* option when creating a composite content
 
 .Repository restrictions
 Docker repositories cannot be included more than once in a composite content view.
-For example, if you attempt to include two content views using the same docker repository in a composite content view, {ProjectServer} reports an error.
+For example, if you attempt to include two content views by using the same docker repository in a composite content view, {ProjectServer} reports an error.
```
Comment (maximiliankolb) [inline guides/common/modules/con_composite-content-views-overview.adoc:55]: alternative:
```suggestion
For example, if you attempt to include two content views with the same docker repository in a composite content view, {ProjectServer} reports an error.
```

or "... that contain ..."

File: guides/common/modules/con_configuring-networking.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@ Use this chapter to configure network services in your integrated {SmartProxy} o
 New hosts must have access to your {SmartProxyServer}.
 {SmartProxyServer} can be either your integrated {SmartProxy} on {ProjectServer} or an external {SmartProxyServer}.
 You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to {ProjectServer} directly, or when the content is synchronized with {SmartProxyServer}.
-Provisioning using the external {SmartProxyServer} can save on network bandwidth.
+Provisioning by using the external {SmartProxyServer} can save on network bandwidth.
```
Comment (maximiliankolb) [inline guides/common/modules/con_configuring-networking.adoc:10]:
```suggestion
Provisioning by using {SmartProxyServers} can save on network bandwidth.
```

out of scope but :see_no_evil:

File: guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc:1]: same as above. please also adjust the file name.

File: guides/common/modules/con_introducing-configuration-management-using-puppet.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_introducing-configuration-management-using-puppet.adoc:1]: as above: please also adjust the file name.

File: guides/common/modules/con_managing-dhcp-using-smart-proxy.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_managing-dhcp-using-smart-proxy.adoc:1]: as above: please adjust the file name too.

File: guides/common/modules/con_managing-dns-using-smart-proxy.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_managing-dns-using-smart-proxy.adoc:1]: same here.

File: guides/common/modules/con_monitoring-hosts-using-red-hat-insights.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_monitoring-hosts-using-red-hat-insights.adoc:1]: same.

File: guides/common/modules/proc_deploying-red-hat-insights-using-the-ansible-role.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_deploying-red-hat-insights-using-the-ansible-role.adoc:1]: too.

**Review by maximiliankolb — COMMENTED** [review]
made some minor suggestions. Please keep the anchor, file name, and title consistent.

File: guides/common/modules/con_composite-content-views-overview.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/con_composite-content-views-overview.adoc:55]:
```suggestion
For example, if you attempt to include two content views that contain the same docker repository in a composite content view, {ProjectServer} reports an error.
```

**Comment by Lennonka** [comment]
Rebased.

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Cherry-picked:
- feb60fa4c9..e9acb59ca7  3.12 -> 3.12

**Pull request merged**